client/components/credit-card/index.jsx

```diff
@@ -0,0 +1,50 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import StoredCard, { cardType } from './stored-card';
+
+class CreditCard extends React.Component {
+	static propTypes = {
+		card: cardType,
+		selected: PropTypes.bool,
+		onSelect: PropTypes.func,
+		className: PropTypes.string,
+	};
+
+	handleKeyPress = event => {
+		if ( event.key === 'Enter' || event.key === ' ' ) {
+			this.props.onSelect( event );
+		}
+	};
+
+	render() {
+		const { card, selected, onSelect, className, children } = this.props;
+		const classes = classNames( 'credit-card', className, { selected, selectable: onSelect } );
```

Comment:
`is-selected` I think is preferable with our [naming standards for modifiers](https://github.com/Automattic/wp-calypso/blob/ea90a8af13d48bbdc774b31543c17b7cb3c6c146/docs/coding-guidelines/css.md#csssass-coding-guidelines), but likely we haven't been super consistent with it.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/24485/commits/dcb4f7fe5e8a9b7cabcac73ef10dc1051e59022c